src/Pages/Tabs.tsx

```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tabs: Tab[]
+};
```

Comment:
Why do you need it as props? You can have tabs constant here in the Tab component